README.md

```diff
@@ -198,7 +227,10 @@ Date[Symbol.builtin] = undefined;
 Builtin.is(Date);                                  // false
 ```
 
-The `Builtin.is()` function will not throw an exception.
+By default, the `Builtin.is()` function will not throw an exception. It is
```

Comment:
It's also possible if it's a getter that throws, but I'm not sure if that warrants mentioning